modules/devShells.nix

```
@@ -17,6 +18,13 @@ mkTransposedPerSystemModule {
       An attribute set of packages to be used as shells.
       [`nix develop .#<name>`](https://nixos.org/manual/nix/stable/command-ref/new-cli/nix3-develop.html) will run `devShells.<name>`.
     '';
+    example = literalExpression ''
+    {
+      devShells.default = pkgs.mkShell {
+          buildInputs = with pkgs; [ terraform wget bat git ];
+        };
+    }
```

The example should be a valid value for the option, whereas this includes the option path leading to the definition.

```suggestion
    # devShells =
    {
      default = pkgs.mkShell {
        nativeBuildInputs = with pkgs; [ terraform wget bat git ];
      };
    }
```